pkg/kubelet/cm/dra/plugin/noderesources.go

```diff
@@ -441,13 +451,27 @@ func (c *nodeResourcesController) sync(ctx context.Context, driverName string) e
 			continue
 		}
 
+		node, err := c.getNode()
+		if err != nil {
+			logger.Error(err, "Creating node resource slice failed")
+			return fmt.Errorf("create node resource slice: %w", err)
```

Comment:
This doesn't describe the error and could lead to a very misleading composed error, depending on what the `err` text is.

```suggestion
			return fmt.Errorf("retrieve node object: %w", err)
```